learning-development/accessibility.qmd

```diff
@@ -4,33 +4,67 @@ title: "Accessibility"
 
 <p class="text-muted">Some quick links and tips for increasing awareness of accessibility in digital content</p>
 
+
+---
+
+## Accessibility in DfE
+
+DfE have our own [Digital Accessibility Hub](https://educationgovuk.sharepoint.com/sites/lvewp00043) filled with helpful information on making the digital world more accessible.
```

Comment:
Potentially add in some context here around why it's important e.g. WCAG (and a link to latest WCAG)

Reply:
Made this change in my PR comments commit